code/modules/clothing/spacesuits/captain.dm

```diff
@@ -13,6 +13,7 @@
 		rad = 50
 	)
 	siemens_coefficient = 0.7
+	style = STYLE_NONE
```

Comment:
if you want to make voidsuits have style, you must make sure account that the Captain always has the most style here, as is everywhere else.